pom.xml

```diff
@@ -331,19 +318,10 @@
           </execution>
         </executions>
         <configuration>
-          <skipITs>${skipIts}</skipITs>
           <disableXmlReport>true</disableXmlReport>
           <useFile>false</useFile>
         </configuration>
       </plugin>
-      <plugin>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <skipTests>${skip.surefire.tests}</skipTests>
-          <runOrder>balanced</runOrder>
```

Comment:
From [surefire documentation](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#runOrder), `balanced` is only meaningful when used with `parallel=classes`. The rest of the plugin configuration is the same as in the plugin-pom.